routes/schemas.js

```diff
@@ -62,6 +62,10 @@ const validatorMessage = Joi.object({
 	isHealthy: Joi.boolean().when('type', {
 		is: 'ApproveState',
 		then: Joi.boolean().required()
+	}),
+	exhausted: Joi.boolean().when('type', {
```

Comment:
only makes sense to have `exhausted` on `NewState` because that's the leader balance (therefore the balance being signed and therefore used)

Reply:
what of this in the issue?
 ```
The problem with that is that NewState is untrusted, so we can add a exhausted boolean to both NewState/ApproveState, and flag the channel if such an event is received from whoami() (i.e. from the sentry's own validator).
```